_shared/project/.github/workflows/deploy.yml

```
@@ -0,0 +1,38 @@
+name: Deploy
+concurrency:
+  group: deploy
+  cancel-in-progress: true
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+jobs:
+  CI:
+    uses: ./.github/workflows/ci.yml
```

```suggestion
  ci:
    uses: ./.github/workflows/ci.yml
    name: Continuous Integration
```

The casing here is a bit strange. It's more obvious on following actions, but I think using a mixed camel case underscore makes it a bit more ambiguous how the file works. Are we linking by id or name? Especially here as we skip the name.

---

:+1: https://github.com/hypothesis/cookiecutters/pull/95/commits/24ed13d0f8d92911f5cd78c71b2eeb028f9a9905

This came about because if a GitHub Actions job doesn't have a `name` then the UI uses the job ID as the display name. So if you can have IDs like `CI`, `QA` and `Production` then you don't need `name`'s and the display names will still look nice (gotta have my capital letters). But it doesn't work for jobs like `Production (Canada)` because you can't have spaces and parens in job IDs. At least not easily, I didn't check whether it can be done with some horrible escaping, especially not because we sometimes need to use a job ID multiple times (when one job depends on another) and because our job IDs live in a JSON file that gets read in and rendered by Jinja2. So in cases like `Production (Canada)` I ended up with IDs like `Production_Canada` and then doing `name: "Production (Canada)"` for display. So you end up with some jobs that have a `name` and some that don't.

I think you're right that it's better to use `lowercase_with_underscores` IDs and always use `name`. It's less weird, more consistent, and makes the template code simpler.